index.html

```diff
@@ -1,276 +1,240 @@
+                  </li>
+                  <li class="media_li"><a href="#" target="_blank"><img src="./images/Medium-icon.svg" alt="Medium icon"/></a>
+                  </li>
+                </ul>
+              </div>
+              <div class="get-b">
+              <a href="#" class="btn btn-mastery ">Get my resume</a>
             </div>
-        </div>
-        <div>
-          <ul>
-            <li aria-expanded="true" aria-haspopup="true">
-              Framework
-             
-              <i class="fa fa-angle-right"></i>
-            </li>
-          </ul>
-           <hr>
-        </div>
-        <div>
-          <ul>
-            <li aria-expanded="true" aria-haspopup="true">
-              Skills
-             
-              <i class="fa fa-angle-right"></i>
-            </li>
-          </ul>
-          <hr>
-        </div>
-      </div>
-    
-
     
-  </section>
-    </div>
+            </div>  
+        <div class="aim">
+          
+          <img src="./images/Skill-List.png" alt="portfolio"/>
+        </div>   
+    </section>
     <footer class="contact_container">
       <div class="contact_bk">
-     <div class="container flex">
-       <h1 class="feedback">Contact me</h1>
-       <p class="msg">
-        if you have an application you are interested in developing,
-        a feature that you need built or a project that needs coding. 
-        I'd love to help with it
-       </p>
-       <div>
-         <form action="https://formspree.io/f/mvodnloy" method="post">
-           <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Rejoice">
-           <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
-           <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
-           <button class="btn" type="submit">Get in Touch</button>
-         </form>
-       </div>
-       <div class="line"></div>
-     </div>
-    </div>
-    </footer>
-  </div> 
-</body>
+        <div class="container">
+          <h1 class="contact_header">Contact me</h1>
+          <p class="contact_p">
+            if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it
+          </p>
+          <div>
+            <form action="https://formspree.io/f/mvodnloy" method="post">
+              <input type="text" aria-required="true" maxlength="30" name="name" placeholder="Patrick Ndimukulaga">
+              <input type="email" aria-required="true" maxlength="30" name="email" placeholder="kapojames@gmail.com">
+              <textarea aria-required="true" maxlength="500" name="message" placeholder="Write your message here"></textarea>
+              <button class="btn" type="submit" >Get in Touch</button>
```

Comment:
- [ ] Kindly follow the button interactions for hover, pressed, and enable as per the Figma design. 

<img width="646" alt="Screenshot 2022-05-05 at 2 27 23 AM" src="https://user-images.githubusercontent.com/70269507/166841293-aef7eb66-6983-4f11-b2b8-1bc68199d3f1.png">